examples/traffic_prediction.jl

```diff
@@ -0,0 +1,96 @@
+# Temporal graph neural networks for traffic prediction
+
+# Load packages
+using Flux
+using Flux.Losses: mae
+using GraphNeuralNetworks
+using MLDatasets: METRLA
+using CUDA
+using Statistics, Random
+CUDA.allowscalar(false)
+
+# Import dataset function
+function getdataset()
+    metrla = METRLA(; num_timesteps = 3)
+    g=metrla[1]
+    features=[]
+    targets=[]
+    graph = GNNGraph(g.edge_index; edata = g.edge_data, g.num_nodes)
+    for i in 1:length(g.node_data.features)
+        push!(features, g.node_data.features[i])
+        push!(targets,g.node_data.targets[i])
+    end
```

Comment:
```suggestion
    features = [x for x in g.node_data.features]
    targets = [y for y in g.node_data.targets]
```
or just directly `features = g.node_data.features`  if that's possible